src/components/ProgressBar/index.tsx

```diff
@@ -248,24 +266,28 @@ const ProgressBar: FunctionComponent<ProgressBarProps> = props => {
     ...props.theme,
   };
   const number = props.percentage;
-  const text = `${!Number.isNaN(number) ? number : '0'}%`;
+  const percentString = `${!Number.isNaN(number) ? number : '0'}%`;
+  const descriptivePercentageString = `${
+    typeof props.circleContents === 'string' ? props.circleContents : null
+  } ${percentString}% Complete`;
+  const textOrPercentString = props.circleContents || percentString;
   return (
     <ThemeProvider theme={theme}>
       <ProgressBarWrapper {...props}>
         {props.isCircular ? (
-          <CircularWrapper percentage={number}>
+          <CircularWrapper percentage={number} circleSize={props.circleSize || DEFAULT_CIRCLE_SIZE}>
             <CircularLeft className="Left">
-              <CircularColorFill />
+              <CircularColorFill barColor={props.barColor} />
             </CircularLeft>
             <CircularRight className="Right">
-              <CircularColorFill />
+              <CircularColorFill barColor={props.barColor} />
             </CircularRight>
-            <CircularValue>{text}</CircularValue>
+            <CircularValue>{textOrPercentString}</CircularValue>
           </CircularWrapper>
         ) : (
           <LineProgressBarWrapper>
-            <LineProgressBar percentage={number > 100 ? 100 : number} />
-            <SROnly>{`${text} Complete`}</SROnly>
+            <LineProgressBar percentage={number > 100 ? 100 : number} barColor={props.barColor} />
+            <SROnly>{descriptivePercentageString}</SROnly>
```

Comment:
feels like we should we have screen reader only bit in the circular graph too?

Reply:
I actually think we should remove it from both. This is because a more detailed description of what the percentage means will be displayed on the page in the circle or by the bar.